docs/howto/troubleshooting.md

```diff
@@ -167,3 +167,27 @@ Please review the [upgrade documentation](../topics/upgrade.md) for a guide on u
 ## Azure API Throttling
 
 See this [document](../topics/azure-api-throttling.md) for help with troubleshooting Kubernetes clusters affected by Azure API throttling.
+
+## Avoid bridge mode problems by using transparent networking
+
+AKS Engine clusters before v0.58.0 or any version with an API model not using "transparent" Kubernetes networking mode may become unavailable if a control plan node becomes `NotReady`. Rebooting the node should recreate a working network configuration, but the problem can be prevented by re-launching clusters using transparent networking instead of bridge mode.

```

Comment:
```suggestion
AKS Engine clusters before v0.58.0 or any version with an API model not using "transparent" Kubernetes networking mode may become unavailable if a control plane node becomes `NotReady`. Rebooting the node should recreate a working network configuration, but the problem can be avoided by provisioning clusters using transparent networking instead of bridge mode.

```